product_docs/docs/pgd/5/proxy/index.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: "EDB Postgres Distributed Proxy"
+navTitle: "PGD Proxy"
+---
+
+EDB Postgres Distributed Proxy is a daemon that acts as an abstraction layer between the client application and Postgres. It interfaces with the BDR consensus mechanism to obtain the identity of the current write leader node and redirects traffic to that node.
+
+There is always at least one global group and one data group in the PGD cluster. BDR elects the write leader for each data group which has the `enable_proxy_routing` and `enable_raft` options set to true. Proxy can be attached to a global group or data group. There could be multiple proxies attached to each group.
+
+PGD Proxy is a TCP layer 4 proxy.
+
+
+## How it works ?
+
+Upon starting, PGD Proxy connects to one of the endpoints given in the local config file and fetches DB connection information for all nodes, proxy options like listen address, listen port, routing details like current write leader, etc. Endpoints given in config file are only used at the time of startup, after that actual endpoints are taken from bdr catalog `bdr.node_routing_config_summary#route_dsn`.
+
+The write leader election is managed by BDR itself. Proxy internally interacts with BDR to get write leader change events notifications on Postgres notify/listen channels and routes client traffic to current write leader. Proxy disconnects all client connections on write leader change. This also applies when `pgd switchover` command is used to invoke a planned transition to a new write leader. Write leader election is a Raft backed activity and is subject to Raft leader availability.
+
+
+## Installing PGD Proxy
+
+### Installing through TPAexec
+
+If PGD cluster is being deployed through TPAexec then it installs and configures PGD Proxy automatically. If you you wish to install PGD Proxy on any other node in PGD cluster then, you simply need to attach the `pgd-proxy` role to that instance in TPAexec's configuration file and set the `bdr_child_group` parameter as shown below before deploying. See [TPAexec](/product_docs/docs/pgd/5/deployments/tpaexec) for more information.
+```yaml
+- Name: proxy-a1
+  location: a
+  node: 4
+  role:
+  - pgd-proxy
+  vars:
+    bdr_child_group: group_a
+  volumes:
+  - device_name: /dev/sdf
+    volume_type: none
+```
+
+
+### Installing manually
```

Comment:
This stuff (also configuration, service file) should go to some subpage as it's not something a normal user should be doing.